src/Extension/ResizeBrowserWindowExtension.php

```diff
@@ -0,0 +1,67 @@
+<?php
+/**
+ * @author    Craig Gosman <craig@ingenerator.com>
+ * @licence   proprietary
```

Comment:
Should be `@license BSD-3-Clause` (or omitted) to match the overall package license. Although I see my IDE settings have put proprietary on all the other files here :)

Reply:
Hadn't spotted it either, mine is doing the same. Could address this in another pull to change the lot as it looks like its everywhere?

Reply:
Yeah good plan. I think best best is just leave the license/licence tag out.